test/e2e/framework/util.go

```diff
@@ -139,7 +139,7 @@ const (
 	PodReadyBeforeTimeout = 5 * time.Minute
 
 	// ClaimProvisionTimeout is how long claims have to become dynamically provisioned.
-	ClaimProvisionTimeout = 5 * time.Minute
+	ClaimProvisionTimeout = 7 * time.Minute
```

Comment:
What if we made this a var so people using this package can just change it

Reply:
good point. Updated